editions/test/tiddlers/tests/data/csv-cases.tid

```diff
@@ -0,0 +1,202 @@
+title: csv-cases
+type: text/plain
+description: A file containing a JSON with test CSVs as string as well as expected results
+
+[
+    {
+        "name": "Simple CSV with no tricks",
+        "options": {},
+        "csv": "cell-11,cell-12,cell-13\r\ncell-21,cell-22,cell-23\r\ncell-31,cell-32,cell-33",
+        "json": [
+            ["cell-11", "cell-12", "cell-13"],
+            ["cell-21", "cell-22", "cell-23"],
+            ["cell-31", "cell-32", "cell-33"]
+        ],
+        "jsonWithHeaders": [
+            {"cell-11": "cell-21", "cell-12": "cell-22", "cell-13": "cell-23"},
+            {"cell-11": "cell-31", "cell-12": "cell-32", "cell-13": "cell-33"}
+        ]
+    },
+    {
+        "name": "Custom separator",
+        "options": {"separator": "\t"},
+        "csv": "cell-11\tcell-12\tcell-13\r\ncell-21\tcell-22\tcell-23\r\ncell-31\tcell-32\tcell-33",
```

Comment:
You could add some "," to the cells' contents, to make it clear that it's no longer a separator.

Reply:
Good idea!